CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Backup] Implement encryption at rest (PR [#6321](https://github.com/vatesfr/xen-orchestra/pull/6321))
```

Comment:
What does "at rest" mean?

Reply:
It means that the backup files are encrypted on the storage.

I believe this terminology is standard but we may need to make it more explicit.